NOTES.md

```diff
@@ -1,3 +1,20 @@
+# Release 2.0.3 (Yojimbo)
+
+Released on 2015-06-10
+
+Bug fix release
+
+* Fixed a bug that broke autocompletion in the Data Explorer (№4261)
```

Comment:
http://en.wikipedia.org/wiki/Numero_sign or Shift-3 on a Russian keyboard layout, which I had accidentally switched back to.